staging/src/k8s.io/apiserver/pkg/endpoints/filters/impersonation.go

```diff
@@ -183,6 +183,23 @@ func WithImpersonation(handler http.Handler, a authorizer.Authorizer, s runtime.
 	})
 }
 
+func userString(u user.Info) string {
+	if u == nil {
+		return "<none>"
+	}
+	b := strings.Builder{}
+	b.WriteString(u.GetName())
```

Comment:
do we have to consider cases where a user name is empty?

Reply:
of course that's not documented in the struct :)